app/views/admin_shelters/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
+    <h3><%= shelter.name %></h3>
+    <p>Created at: <%= shelter.created_at %></p>
+  </div>
+<% end %>
+
+<h3>Shelters with pending applications</h3>
+<% @shelters_list.each do |shelter|  %>
```

Comment:
See my comment in your `shelters_controller` but in here is where you would want to have created a model method to get shelters with pending apps and call that method here vs creating an instance method in your controller. So something like: 
```ruby
@shelters.with_pending_applications.each do
```